packages/dlp/package.json

```diff
@@ -1,12 +1,16 @@
 {
   "repository": "GoogleCloudPlatform/google-cloud-node",
-  "name": "dlp",
-  "version": "0.0.0",
+  "name": "@google-cloud/dlp",
+  "version": "0.7.1",
```

Comment:
@swcloud We can actually overwrite it on a per api basis.

https://github.com/googleapis/googleapis/pull/357